backend/pkg/api/routes.go

```diff
@@ -207,7 +207,14 @@ func (api *API) routes() *chi.Mux {
 			r.Handle("/admin/metrics", promhttp.Handler())
 			r.Handle("/admin/health", api.handleLivenessProbe())
 			r.Handle("/admin/startup", api.handleStartupProbe())
+		})
 
+		// Debug routes
+		router.Group(func(r chi.Router) {
+			api.Hooks.Route.ConfigInternalRouter(r)
+			if api.Cfg.REST.Debug.ForceLoopback {
+				r.Use(forceLoopbackMiddleware(api.Logger))
+			}
```

Comment:
Wondering if we should check whether debug endpoints are enabled at all first. The enabled check could even be outside line 213 where we create the router group no?